homeassistant/components/bmw_connected_drive/config_flow.py

```diff
@@ -132,6 +143,12 @@ async def async_step_account_options(
                         CONF_READ_ONLY,
                         default=self.config_entry.options.get(CONF_READ_ONLY, False),
                     ): bool,
+                    vol.Required(
+                        CONF_SCAN_INTERVAL,
+                        default=self.config_entry.options.get(
+                            CONF_SCAN_INTERVAL, DEFAULT_SCAN_INTERVAL_MINUTES
+                        ),
+                    ): vol.All(int, vol.Range(min=5)),
```

Comment:
We don't allow for configurable scan intervals. Home Assistant should ship with a sane default scan interval out of the box.

If users want different intervals, they can use an automation using the `homeassistant.update_entity` service to update according to their own schedule and speed.

Reply:
Thanks for the quick reply! In that case, I'll close this PR as we don't have sensible defaults yet after the the recent introduction of quotas on our APIs.